src/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  cursor: pointer;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--5 {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
apply styles for all modifiers (1-5). Just reuse your solution from a previous task